src/hooks/useBreadcrumbs.ts

```diff
@@ -1,18 +1,20 @@
-import { useMemo } from 'react';
+import { ComponentType, useMemo } from 'react';
 import { useUrlParams } from '.';
 import { buildChallengeUrl, buildHubUrl, buildOpportunityUrl } from '../utils/urlBuilders';
 import { useChallengeNameQuery, useHubNameQuery, useOpportunityNameQuery } from './generated/graphql';
+import HubIcon from '@mui/icons-material/Hub';
+import FilterHdrIcon from '@mui/icons-material/FilterHdr';
```

Comment:
Use the same icon as on the top-level menu for Challenges (with the flag).

Reply:
I talked with Simone about this, I think we agreed to use a simpler icon in here because the one with the flag is too complex and looks weird when printed so small. But will confirm tomorrow with her.